### PR TITLE
Revise files header license

### PR DESCRIPTION
**Pull request by malukenho**
I added a `docheader` checker to match the license on files. If it's overkill just tell me, so I can remove it.

### CODE REVIEW

File: .docheader [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*
+ * This file is part of the prooph/service-bus.
+ * (c) 2014-%year% prooph software GmbH <contact@prooph.de>
```
Comment (prolic) [inline .docheader]: add here another line:

    (c) 2014-%year% Sascha-Oliver Prolic <saschaprolic@googlemail.com>

It was on our todo list for a long time to update to license headers :)

File: .travis.yml [hunk 2]
```diff
@@ -17,6 +17,7 @@ before_script:
 script:
   - php ./vendor/bin/phpunit --coverage-text --coverage-clover ./build/logs/clover.xml
   - ./vendor/bin/php-cs-fixer fix -v --diff --dry-run
+  - ./vendor/bin/docheader check src/
```
Comment (prolic) [inline .travis.yml]: can we also add tests to check?

**Comment by prolic** [comment]
I am good with the change. But can you do it to develop branch? We are about to release a new minor version from develop-branch, soon.

**Comment by codeliner** [comment]
I like the changes, too. thx @malukenho

**Comment by prolic** [comment]
Manually merged into develop and applied needed improvements.

**Pull request closed without merging**